ShopperInsights/src/main/java/com/braintreepayments/api/EligiblePaymentsApi.kt

```diff
@@ -1,28 +1,34 @@
 package com.braintreepayments.api
 
 import com.braintreepayments.api.EligiblePaymentsApiRequest.Companion.toJson
+import org.json.JSONException
 
-internal class EligiblePaymentsApi {
-    fun execute(request: EligiblePaymentsApiRequest): EligiblePaymentsApiResult {
-        request.toJson()
-        // TODO: Network call
-
-        // Hardcoded result
-        return EligiblePaymentsApiResult(
-            eligibleMethods = EligiblePaymentMethods(
-                paypal = EligiblePaymentMethodDetails(
-                    canBeVaulted = true,
-                    eligibleInPayPalNetwork = true,
-                    recommended = true,
-                    recommendedPriority = 1
-                ),
-                venmo = EligiblePaymentMethodDetails(
-                    canBeVaulted = true,
-                    eligibleInPayPalNetwork = true,
-                    recommended = true,
-                    recommendedPriority = 1
-                )
-            )
+internal class EligiblePaymentsApi(
+    private val braintreeClient: BraintreeClient
+) {
+    fun execute(request: EligiblePaymentsApiRequest, callback: EligiblePaymentsCallback) {
+        val jsonBody = request.toJson()
+        // TODO: Move url to PaypalHttpClient class when it is created
+        val url = "https://api.sandbox.paypal.com/v2/payments/find-eligible-methods"
```

Comment:
This URL should be able to switch b/w `api.sandbox.paypal.com` & `api.paypal.com` based on the env of the ClientToken provided to initialize the `BraintreeClient`. We get the env from Config lookup call [here](https://github.com/braintree/braintree_android/blob/13140a52d349533d1285d7c0a0257852c3f1cd57/BraintreeCore/src/main/java/com/braintreepayments/api/ConfigurationLoader.kt#L16) (see [Configuration.environment ](https://github.com/braintree/braintree_android/blob/13140a52d349533d1285d7c0a0257852c3f1cd57/BraintreeCore/src/main/java/com/braintreepayments/api/Configuration.kt#L80)property)

Reply:
Ah, I did not see that. Thank you. Fixed 8c921affcfcfcf64f2fbce2c9c0787cbe6358595